gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/GqlQuery.java

```diff
@@ -212,6 +212,11 @@ static Binding fromPb(com.google.datastore.v1beta3.GqlQueryParameter argPb) {
       namedBindings.put(name, toBinding(DateTimeValue.MARSHALLER, Arrays.asList(value)));
       return this;
     }
+    
+    public Builder<V> setBinding(String name, GeoPoint... value) {
```

Comment:
For here and in Structured Query. I don't think GeoPoint can be used in queries (for now). Did you see any documentation suggesting otherwise?